test/export/test_export.py

```diff
@@ -799,6 +799,44 @@ def forward(self, x, weight, bias):
                 actual_result.append(node.meta.get("torch_fn"))
         self.assertEqual(actual_result, expected_result)
 
+    def test_hoo_inline_users_issue(self):
```

Comment:
test_hop?

Reply:
higher order op? I have no idea what acronym is the right one hahaha